### PR TITLE
Fixes and improvements to Semantic analysis Get started

### PR DESCRIPTION
**Pull request by svick**
cc: @BillWagner

### CODE REVIEW

File: docs/csharp/roslyn-sdk/get-started/semantic-analysis.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ In this tutorial, you explore the **Symbol** and **Binding APIs**. These APIs pr
...
+An instance of <xref:Microsoft.CodeAnalysis.Compilation> is analogous to a single project as seen by the compiler and represents everything needed to compile a Visual Basic or C# program. The **compilation** includes the set of source files to be compiled, assembly references, and compiler options. You can reason about the meaning of the code using all the other information in this context. A <xref:Microsoft.CodeAnalysis.Compilation> allows you to find **Symbols** - entities such as types, namespaces, members, and variables which names and other expressions refer to. The process of associating names and expressions with **Symbols** is called **Binding**.
```
Comment (svick) [inline docs/csharp/roslyn-sdk/get-started/semantic-analysis.md]: What changed here is added "and" in "assembly references, and compiler options".

(GitHub doesn't highlight this change.)

Comment (BillWagner) [reply docs/csharp/roslyn-sdk/get-started/semantic-analysis.md]: Thanks @svick (BTW, the "rich diff" button at the top of each diff will highlight changes like these.

File: samples/csharp/roslyn-sdk/SemanticQuickStart/Program.cs [hunk 2]
```diff
@@ -64,7 +58,7 @@ static void Main(string[] args)
             var systemSymbol = (INamespaceSymbol)nameInfo.Symbol;
             foreach (INamespaceSymbol ns in systemSymbol.GetNamespaceMembers())
             {
-                Console.WriteLine(ns.Name);
+                Console.WriteLine(ns);
```
Comment (BillWagner) [inline samples/csharp/roslyn-sdk/SemanticQuickStart/Program.cs]: I'm curious why you made this change. I liked showing the distinction between `ns.ToString()` which returns the flly qualified name, and `ns.Name` which returns the scoped name. It's not a strong preference, but I'm curious.

**Review by BillWagner — COMMENTED** [review]
This looks great @svick I had one comment on a change. I'd like your thoughts.

File: samples/csharp/roslyn-sdk/SemanticQuickStart/Program.cs (hunk 2 above)
Comment (svick) [reply samples/csharp/roslyn-sdk/SemanticQuickStart/Program.cs]: I think that knowing how to get the full namespace name is more useful and less discoverable than getting just the scoped name.

Also, I think it makes the output clearer: when I see the list `System.Collections`, `System.Configuration`, `System.Deployment`, I can clearly see those are namespace. When I saw `Collections`, `Configuration`, `Deployment`, it took me a second to figure out what I was looking at.

For those reasons, I prefer it this way, but it's not a very strong preference either.

**Review by BillWagner — APPROVED** [review]
Thanks for making these updates @svick 

I'll :shipit: now.  

Thanks again for your continued participation and contributions.

**Pull request merged**